src/test/scala/tech/cryptonomic/conseil/metadata/TransparentUnitTransformation.scala

```diff
@@ -0,0 +1,26 @@
+package tech.cryptonomic.conseil.metadata
+
+import tech.cryptonomic.conseil.config.MetadataConfiguration
+import tech.cryptonomic.conseil.generic.chain.PlatformDiscoveryTypes
+
+class TransparentUnitTransformation(overrides: MetadataConfiguration = MetadataConfiguration(Map.empty))
```

Comment:
What about some very basic scaladoc here?

Reply:
Also, I guess we can use an object here

```scala
object TransparentUnitTransform extends UnitTransform(MetadataConfiguration(Map.empty) {
  ...
}
```

Reply:
done